src/Tickets/Emails/Email/Completed_Order.php

```diff
@@ -0,0 +1,357 @@
+	 * Get email settings.
+	 *
+	 * @since TBD
+	 *
+	 * @return array
+	 */
+	public function get_settings(): array {
+
+		$settings = [
+			[
+				'type' => 'html',
+				'html' => '<div class="tribe-settings-form-wrap">',
+			],
+			[
+				'type' => 'html',
+				'html' => '<h2>' . esc_html__( 'Completed Order Email Settings', 'event-tickets' ) . '</h2>',
+			],
+			[
+				'type' => 'html',
+				'html' => '<p>' . esc_html__( 'The site admin will receive an email about any orders that were made. Customize the content of this specific email using the tools below. The brackets {event_name}, {event_date}, and {ticket_name} can be used to pull dynamic content from the ticket into your email. Learn more about customizing email templates in our Knowledgebase.' ) . '</p>',
+			],
+			static::$option_enabled => [
+				'type'                => 'toggle',
+				'label'               => esc_html__( 'Enabled', 'event-tickets' ),
+				'default'             => true,
+				'validation_type'     => 'boolean',
+			],
+			static::$option_recipient => [
+				'type'                => 'text',
+				'label'               => esc_html__( 'Recipient(s)', 'event-tickets' ),
+				'default'             => $this->get_default_recipient(),
+				'size'                => 'large',
+				'validation_type' => 'email_list',
+			],
+			static::$option_subject => [
+				'type'                => 'text',
+				'label'               => esc_html__( 'Subject', 'event-tickets' ),
+				'default'             => $this->get_default_subject(),
+				'placeholder'         => $this->get_default_subject(),
+				'size'                => 'large',
+				'validation_callback' => 'is_string',
+			],
+			static::$option_heading => [
+				'type'                => 'text',
+				'label'               => esc_html__( 'Heading', 'event-tickets' ),
+				'default'             => $this->get_default_heading(),
+				'placeholder'         => $this->get_default_heading(),
+				'size'                => 'large',
+				'validation_callback' => 'is_string',
+			],
+			static::$option_add_content => [
+				'type'                => 'wysiwyg',
+				'label'               => esc_html__( 'Additional content', 'event-tickets' ),
+				'default'             => '',
+				'tooltip'             => esc_html__( 'Additional content will be displayed below the tickets in your email.', 'event-tickets' ),
+				'validation_type'     => 'html',
+				'settings'        => [
+					'media_buttons' => false,
+					'quicktags'     => false,
+					'editor_height' => 200,
+					'buttons'       => [
+						'bold',
+						'italic',
+						'underline',
+						'strikethrough',
+						'alignleft',
+						'aligncenter',
+						'alignright',
+					],
+				],
+			],
+		];
+
+		/**
+		 * Allow filtering the settings for this email.
+		 *
+		 * @since TBD
+		 *
+		 * @param array          $settings  The settings array.
+		 * @param Email_Abstract $this      Email object.
+		 */
+		return apply_filters( 'tec_tickets_emails_settings_' . self::$id, $settings, $this );
```

Comment:
Rather than using a variable in the filter names, I went ahead and just added each of the email names in so they made sense. For example, this one you pointed out is now `tec_tickets_emails_completed_order_settings`